### ISSUE TITLE
Onshape provider endpoints are not up to date

### ISSUE THREAD

**Issue by koliyo** [aspnet-contrib/AspNet.Security.OAuth.Providers#469]
**Describe the bug**
When using the `AspNet.Security.OAuth.Onshape` provider with the default endpoint values I get `invalid_grant` response.

See https://github.com/aspnet-contrib/AspNet.Security.OAuth.Providers/blob/dev/src/AspNet.Security.OAuth.Onshape/OnshapeAuthenticationDefaults.cs

By using these updated endpoint declarations I get proper auth flow:
```cs
        .AddOnshape(options => {
          IConfigurationSection outhConfig =
              Configuration.GetSection("Authentication:Onshape");

          options.ClientId = outhConfig["ClientId"];
          options.ClientSecret = outhConfig["ClientSecret"];
          options.UserInformationEndpoint = "https://cad.onshape.com/api/users/sessioninfo";
          options.AuthorizationEndpoint = "https://oauth.onshape.com/oauth/authorize";
          options.TokenEndpoint = "https://oauth.onshape.com/oauth/token";
        })
```

ie `cad.onshape.com` -> `oauth.onshape.com`

**Comment by martincostello** [comment]
Thanks for letting us know @koliyo - I'll look at updating the provider and pushing a new version later today or tomorrow 👍

**Comment by martincostello** [comment]
HI @koliyo - an updated version of the package, `3.1.4-preview.20425.44`, is now available from our [MyGet feed](https://www.myget.org/feed/aspnet-contrib/package/nuget/AspNet.Security.OAuth.Onshape/3.1.4-preview.20425.44).

Would you be able to try it out in your application and verify it resolves your need to override the URLs on the options?

Now you should be able to just go back to:

```cs
.AddOnshape(options => {
    IConfigurationSection outhConfig =
        Configuration.GetSection("Authentication:Onshape");

    options.ClientId = outhConfig["ClientId"];
    options.ClientSecret = outhConfig["ClientSecret"];
})
```

**Comment by martincostello** [comment]
Hi @koliyo - were you able to validate the fix for the Onshape provider?

**Comment by martincostello** [comment]
As you've not provided any feedback I'm going to assume it works.

**Comment by martincostello** [comment]
This fix is now available in version [3.1.4](https://www.nuget.org/packages/AspNet.Security.OAuth.Onshape/3.1.4) of the provider.
